packages/flutter/test/widgets/text_selection_test.dart

```diff
@@ -380,4 +405,213 @@ void main() {
 
     await gesture.removePointer();
   });
+
+  testWidgets('test EditableTextSelectionHandlersProvider long press', (WidgetTester tester) async {
+    await pumpEditableTextSelectionHandlersProvider(tester);
+    final TestGesture gesture =
+      await tester.startGesture(const Offset(200.0, 200.0), pointer: 0, kind: PointerDeviceKind.touch);
```

Comment:
Where does 200,200 come from?

Reply:
I copy from other test, and it seems to work fine.